docs/docker_linux.rst

```diff
@@ -0,0 +1,50 @@
+Use LISA docker image on Linux
+=====================
+
+This guide will walk you through using the LISA Docker image on a Linux system.
+
+Install docker on Linux
+-----------------------
+
+The first steps are to install Docker on your system.
+
+On Ubuntu, you can install Docker with the following commands:
+
+.. code:: bash
+
+   sudo apt update
+   sudo apt install docker.io -y
+
+On Azure Linux, you can install Docker with the following commands:
+
+.. code:: bash
+
+   sudo tdnf update
+   sudo tdnf install -y moby-engine moby-cli
+
+Start Docker service
+--------------------
+
+After installing Docker, you can start the Docker service with the following command:
+
+.. code:: bash
+
+   sudo systemctl start docker
+
+List docker images
+------------------
+
+You can check the available tags for the LISA Docker image by listing the tags from the Microsoft Azure Registry.
```

Comment:
This step is not required. Just remove all unnecessary steps, so make the whole process easy. We ask people to use latest. If they want to use version specified, they can figure it out.